src/components/ProposalTable.tsx

```diff
@@ -21,29 +26,49 @@ interface ProposalTableProps {
 export const ProposalTable = ({
   values,
   version,
-}: ProposalTableProps) => (
-  <Table>
-    <TableHead fixed>
-      <TableRow>
-        <ColumnHead>
-          Canonical Field
-        </ColumnHead>
-        <ColumnHead>
-          {`Version ${version}`}
-        </ColumnHead>
-      </TableRow>
-    </TableHead>
-    <TableBody>
-      {values.map(({ fieldName, position, value }) => (
-        <TableRow key={position}>
-          <RowHead>
-            {fieldName}
-          </RowHead>
-          <RowCell>
-            {value}
-          </RowCell>
+}: ProposalTableProps) => {
+  const [displayShortCodes, setDisplayShortCodes] = useState(false);
+
+  const handleDisplayShortCodesClick = () => {
+    setDisplayShortCodes((previous) => !previous);
+  };
+
+  return (
+    <Table>
+      <TableHead fixed>
+        <TableRow>
+          <ColumnHead actions>
+            Canonical Field
+            <ColumnActions>
+              <ColumnAction
+                title="Toggle between field label and short code"
```

Comment:
One small design idea that I think I got from conflating this functionality with the version history functionality:

What if this button hid or revealed an extra column, instead of swapping out columns? I think right now our canonical field "short" codes are Very Similar to their labels, but that might not always be the case.

Reply:
I'm going to consider this, but leave it as-is for now. Toggling is quick and can be done from any scroll point, and I've thought of this as sort of a quick reference. I'd rather reserve the horizontal real estate, which will soon be narrowed by the sidebar and potentially the [source label](https://github.com/orgs/PhilanthropyDataCommons/projects/2/views/1?pane=issue&itemId=21563803).